tests/PSRule.Rules.Azure.Tests/Azure.ContainerApp.Tests.ps1

```diff
@@ -85,6 +85,22 @@ Describe 'Azure.ContainerApp' -Tag 'ContainerApp' {
             $ruleResult | Should -Not -BeNullOrEmpty;
             $ruleResult.Length | Should -Be 1;
             $ruleResult.TargetName | Should -BeIn 'capp-env-C';
+
+        It 'Azure.ContainerApp.ExternalIngress' {
+            $filteredResult = $result | Where-Object { $_.RuleName -eq 'Azure.ContainerApp.ExternalIngress' };
```

Comment:
On line 37 you will need to add:

```powershell
$result = Invoke-PSRule @invokeParams -InputPath $dataPath -Option @{
    'Configuration.AZURE_CONTAINERAPPS_RESTRICT_INGRESS' = $True
}
```

Otherwise this rule will not execute and you should get a test failure.